engine/src/flutter/shell/platform/embedder/embedder.h

```diff
@@ -1594,6 +1628,9 @@ typedef struct {
   // Array of string attributes associated with the `decreased_value`.
   // Has length `decreased_value_attribute_count`.
   const FlutterStringAttribute** decreased_value_attributes;
+  // The set of semantics flags associated with this node. Prefer to use this
+  // over `flags`.
+  FlutterSemanticsFlags* flags2;
```

Comment:
I vaguely remember we have to be careful about using pointer where the value structure may expand in the future, but i can't remember exactly what may have gone wrong if we don't. cc @loic-sharma do you think this will be ok? We are going to add more bool to the struct FlutterSemanticsFlags in the future.